llvm/test/CodeGen/AArch64/GlobalISel/store-merging.mir

```diff
@@ -586,8 +594,6 @@ body:             |
   bb.1 (%ir-block.0):
     liveins: $x0, $x1
 
-    ; The store to ptr2 prevents merging into a single store.
-    ; We can still merge the stores into addr1 and addr2.
```

Comment:
Should these comments be removed?

Reply:
No but I can't seem to get the update script to keep them.

Reply:
You can try to move the comment above the `...` .